### ISSUE TITLE
The signal scaling in the training stage

### ISSUE THREAD

**Issue by pengxinliu** [ShoufaChen/DiffusionDet#9]
Hi, 

Thanks for sharing your wonderful work. I can't understand the signal scaling equation `pb = (pb * 2 - 1) * scale` in the training stage, can you explain the reason why transform `pb` by this equation in more detail? 

Many thanks!

**Comment by ShoufaChen** [comment]
Hi,

Thanks for your interest in our work.

The raw signals~(eg, image pixel, ground truth boxes coordinates) should be scaled to some range before combination with noise. Taking image generation, for example, raw pixel values are typically scaled to [-1, 1]. 

We also conduct ablation studies on this scaling value in Table 3 (a) in our manuscript. A more detailed discussion can be found in Sec 4.4 **Signal scaling**.

**Comment by kaneyxx** [comment]
I have the same question. According to your references in this part, "Analog Bits" and panoptic paper convert the input data to bits signal. Here you just normalized the ground truth info to [0, 1] then conduct the scale. Is it the same concept or you just want to do the same operation? I think the scale is just to translate the center and scale the height and width of boxes. Please help me to figure out this. Thank you ~!

**Comment by aoyanl** [comment]
I also have a question about signal scaling,  the codes below are added after the combination of signal and noise,
 x = torch.clamp(x, min=-1 * self.scale, max=self.scale)
 x = ((x / self.scale) + 1) / 2.
can you explain why need to scale the noisy mask again？
